<launch>
    <param name="pos_x0" value="0" />
    <param name="pos_y0" value="0" />
    <param name="theta0" value="0" />

    <node pkg="project1" type="odometry" name="odometry" output = "screen" />
    <node pkg="project1" type="wheel_speed" name="wheel_speed" output="screen" />
    <node pkg="project1" type="tf2_broad" name="tf2_broad" />

    <!-- tf for 1st bag -->
    <node pkg="tf2_ros" type="static_transform_publisher" name="world"
          args="0.00815962441266 0.003059721319 0.368708342314 
                -0.0115776332095 -0.0207516606897 -0.0195951275527 0.999525606632 world odom" />
 
    <!-- tf for 2nd bag 
    <node pkg="tf2_ros" type="static_transform_publisher" name="world"
          args="0.0242218170315 0.00374252465554 0.372348427773 
                0.0338488332927 -0.0555740334094 -0.00280516617931 0.997876763344 world odom" />
    -->

    <!-- tf for 3rd bag
    <node pkg="tf2_ros" type="static_transform_publisher" name="world"
          args="0.00546012399718 0.0399703718722 0.370361357927 
                -0.0105932056904 -0.0247681923211 -0.00675158947706 0.999614357948 world odom" />
    -->

</launch>